src/ml_board/frontend/Dashboard/src/components/experimentPage/HalfDonoughtGraph/HalfDonoughtGraph.tsx

```diff
@@ -0,0 +1,74 @@
+
+import { Doughnut } from 'react-chartjs-2';
+import styles from './HalfDonoughtGraph.module.css';
+import { Chart as ChartJS, Filler, ArcElement, Tooltip, Legend } from 'chart.js';
+ChartJS.register(
+    ArcElement,
+    Tooltip,
+    Legend,
+    Filler
+);
+
+interface HalfDonoughtGraphDataProps {
+    graph_data: Array<number>;
+    label_data: Array<string>;
+}
+
+const HalfDonoughtGraph: React.FC<HalfDonoughtGraphDataProps> = (props) => {
+    
+    return (
+        <Doughnut
+            className={styles.donought}
+            data={{
+                labels: props.label_data,
+                datasets: [
+                    {
+                        data: props.graph_data,
+                        backgroundColor: [
+                            "#36A2EB",
+                            "transparent"
+                        ],
+                        borderColor: "#D1D6DC",
+                        borderWidth: 1,
+                    },
+                ],
+            }}
+            options={{
+                animation: {
+                    duration: 0,
+                    easing: 'linear'
+                },
+                plugins: {
+                    legend: {
+                        display: false
+                    },
+                    tooltip: {
+                        enabled: false
+                    }
+                },
+                rotation: -90,
+                circumference: 180,
+                cutout: "70%",
+                maintainAspectRatio: true,
+                responsive: true
+            }}
```

Comment:
do we need the CSS here or can we move it to a dedicated CSS file?

Reply:
I tried to find a way, but I couldn't find any for now. So I think we should proceed as @OsamaMSoliman mentioned

Reply:
> this has to do with **chart js** and their implementation, I suggest for now accepting the PR and opening a separate issue for all the css related to **chart js**! because this is not the only place where we have it!

@OsamaMSoliman could you create an issue for this then please?